### PR TITLE
Update update_dashboard.yml

### LINKED ISSUE #153: Update twice daily
Have the code run twice daily so as to have updated ward numbers at the end of the day

### PR DESCRIPTION
**Pull request by Nozziel**
closes #153

### CODE REVIEW

File: .github/workflows/update_dashboard.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name: Update Covid 19 dashboard
 on:
   schedule:
     - cron: '0 22 * * *' # 22:00 Utc with the aim to be finished at 24:00 utc in 2 hours (5:30am India [IST])
+    - cron: '0 10 * * *' # 10:00 Utc 15:30 India [IST] a second run to catch updated wards
```
Comment (aswinjayan94) [inline .github/workflows/update_dashboard.yml:6]: Can we change this to 13:00 UTC (18:30 IST)? I'm not sure if the PDF gets updated by 15:30 IST..

**Review by aswinjayan94 — CHANGES_REQUESTED** [review]
Let's perform the second run at 13:00 UTC (this time should ensure that the ward PDFs are updated with the previous day's data)

**Comment by Nozziel** [comment]
> Let's perform the second run at 13:00 UTC (this time should ensure that the ward PDFs are updated with the previous day's data)

done

File: .github/workflows/update_dashboard.yml (hunk 1 above)
Comment (Nozziel) [reply .github/workflows/update_dashboard.yml]: done

**Review by Nozziel — COMMENTED** [review]
ok

**Review by aswinjayan94 — APPROVED** [review]
Looks good

**Pull request merged**